sdk/template/template-dpg/package.json

```
@@ -0,0 +1,138 @@
+{
+  "name": "@azure/template-dpg",
+  "version": "1.0.0-beta.1",
+  "description": "Widget Service",
+  "sdk-type": "client",
+  "main": "dist/index.js",
+  "module": "dist-esm/src/index.js",
+  "react-native": {
+    "./dist/index.js": "./dist-esm/src/index.js"
+  },
+  "type": "module",
+  "exports": {
+    ".": {
+      "types": "./types/src/index.d.ts",
+      "require": "./dist/index.cjs",
+      "import": "./dist-esm/src/index.js"
+    },
+    "./api": {
+      "types": "./types/src/api/index.d.ts",
+      "import": "./dist-esm/src/api/index.js"
+    },
+    "./rest": {
+      "types": "./types/src/rest/index.d.ts",
+      "import": "./dist-esm/src/rest/index.js"
+    }
+  },
+  "//metadata": {},
+  "types": "types/latest/template-dpg.d.ts",
+  "scripts": {
+    "audit": "node ../../../common/scripts/rush-audit.js && rimraf node_modules package-lock.json && npm i --package-lock-only 2>&1 && npm audit",
+    "build:samples": "echo Obsolete",
+    "build:test:browser": "tsc -p . && rollup -c rollup.test.config.js 2>&1",
```

Do we need the custom rollup config?
```suggestion
    "build:test:browser": "tsc -p . && dev-tool run bundle",
```